templates/vue/src/components/Lightbox/media/ActivityMedia/Question/index.vue

```diff
@@ -120,6 +144,22 @@ export default {
     isLoggedIn() {
       return wp.isLoggedIn()
     },
+    hasPreviousSubmission() {
+      return Object.keys(this.answers).length > 0
+    },
+    isEditable() {
+      return this.question.followUp.allowAnswerEdit
```

Comment:
We should not be storing this under the followup for a question as that is not what we are concerned with here. This is for ANSWER nodes and so it should be saved as a property of `typeData` (maybe called `isEditable`)